db/schema.rb

```diff
@@ -50,6 +50,11 @@
     t.datetime "remember_created_at"
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
+    t.string "confirmation_token"
+    t.string "unconfirmed_email"
```

Comment:
Are we using `reconfirmable` ?

Reply:
Based on [this line](https://github.com/rootstrap/rails_hotwire_base/blob/main/config/initializers/devise.rb#L160) I think we are, right?

Reply:
Seems like it 😄